README.md

```diff
@@ -114,7 +114,7 @@ Hibernate or JPA.
 
 Add the following dependency to your project:
 
-	org.hibernate.rx:hibernate-rx-core:1.0.0-SNAPSHOT
+	org.hibernate.reactive:hibernate-rx-core:1.0.0-SNAPSHOT
```

Comment:
should the artifact IDs be changed to match also? e.g. `hibernate-reactive-core`

Reply:
My 2c is that if the group ID is exactly identical to the artifact id, then we have a redundancy and what the hell are artifact ids even for?